app/views/providers/check_provider_answers/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_with(model: check_provider_answers, local: true) do |form| %>
```

Comment:
❓ I can't see this template being used anywhere. Is it actually required? If not, let's ✂️ it!

Reply:
👍